tdrs-backend/tdpservice/users/migrations/0012_add_initial_super_user.py

```diff
@@ -0,0 +1,25 @@
+import os
+from django.db import migrations
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('users', '0011_auto_20210108_1741'),
+    ]
+
+    def generate_superuser(apps, schema_editor):
+        User = apps.get_model('users', 'User')
+
+        # set the environment variable to the username of the
+        # initial superuser
+        su_username = os.environ.get('DJANGO_SU_NAME', 'admin')
+
+        superuser = User.objects.create_superuser(
```

Comment:
So the test failures we're seeing now are because of this except described in the [Django docs](https://docs.djangoproject.com/en/3.1/topics/migrations/#historical-models)
> Because it’s impossible to serialize arbitrary Python code, these historical models will not have any custom methods that you have defined. They will, however, have the same fields, relationships, managers (limited to those with use_in_migrations = True) and Meta options (also versioned, so they may be different from your current ones).
Since `create_superuser` is defined on the `UserManager` and that has `use_in_migrations=True` it is available to the historical model, however during that method `normalize_username` is called which exists as a model method on `AbstractUser`.

Since the historical model doesn't serialize those functions we can't use the `create_superuser` outright with historical models in migrations. As an alternative, we can mimic the functionality of the `create_superuser` call but instead go straight to the `User.objects.create` method.

The migration below utilizes historical models and creates a superuser the same way, without causing any test failures:
```
import os
from django.contrib.auth.hashers import make_password
from django.db import migrations
from django.utils import timezone

class Migration(migrations.Migration):
    dependencies = [
        ('users', '0011_auto_20210108_1741'),
    ]

    def generate_superuser(apps, schema_editor):
        # Use the historical model to prevent this from failing on clean
        # builds if the User model changes in the future
        User = apps.get_model('users', 'User')

        # set the environment variable to the username of the
        # initial superuser
        su_username = os.environ.get('DJANGO_SU_NAME', 'admin')

        # Get current time for date_joined
        now = timezone.now()

        # Sets a password the user won't be able to log in with
        # Needed because we defer to Login.gov for authentication
        # and users mustn't be allowed to login directly.
        unusable_password = make_password(None)

        # Note we manually call the `create` method instead
        # This allows us to operate on Historical Models, which
        # do not have access to arbitrary model methods.
        # https://docs.djangoproject.com/en/3.1/topics/migrations/#historical-models  noqa
        superuser = User.objects.create(
            username=su_username,
            email=su_username,
            date_joined=now,
            is_active=True,
            is_staff=True,
            is_superuser=True,
            password=unusable_password
        )

        superuser.save()

    operations = [
        migrations.RunPython(generate_superuser),
    ]

```